portal-web/test/functional/com/liferay/portalweb/macros/Clustering.macro

```diff
@@ -272,6 +272,25 @@ definition {
 		}
 	}
 
+	macro waitForNodeDeactivation {
+		var nodeLog = Clustering.getLogFileSpecificServer(nodePort = ${nodePort});
+
+		var minutestoExpire = RegexUtil.replace(${nodeLog}, "temporarily permitted during the grace period which expires in (.*[0-9]) minutes", 1);
+
+		var mstoExpire = ${minutestoExpire} * 60000;
+
+		var mstoExpire = ${mstoExpire} + 60000;
+
+		var javascript = '''
```

Comment:
What is the purpose of this? Isnt this just converting an integer to a string and then back to an integer? Cant we just remove this block and it performs the same behavior?